kernel/update_kernel_fork.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
```

Comment:
Again, this file needs a copyright header.

Reply:
done